packages/core/docs/changelog/6347.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  description: 'getProduct custom query - productType doesnt get mapped to product response in enhanceProduct helper',
+  link: 'https://github.com/vuestorefront/vue-storefront/issues/6347',
+  isBreaking: false,
+  author: 'Marcin Sulowski',
+  linkToGitHubAccount: 'https://github.com/MarcinSulowski'
+};
```

Comment:
Because this is commercetools-only change, please move this file to `packages/core/docs/commercetools/changelog`.

Reply:
will do, thanks!